src/util/testutils/client.js

```diff
@@ -0,0 +1,7 @@
+export const fakeClient = {
+  search: jest.fn(requests =>
```

Comment:
Prefer the usage of a factory otherwise we share the same `jest.fn` for all the tests.